README.md

```diff
@@ -58,7 +58,7 @@ MC_MID="<YOUR_SFMC_MID>"
 MC_SERVER_URL="<YOUR_SFMC_URL>"
 ```
 ## Setup iOS 
-
+### Update `info.plist`
```

Comment:
Move it to line 64, and replace it with '### Provision for Push'.

Reply:
Please update src/README.md file as well. thanks
README.md and src/README.md should be equal.

Reply:
fixed

Reply:
My first comment is not resolved, I will change it by myself.